e2e-tests/conf/minio-secret.yml

```diff
@@ -3,5 +3,5 @@ kind: Secret
 metadata:
   name: minio-secret
 stringData:
-  AWS_ACCESS_KEY_ID: some-access-key
-  AWS_SECRET_ACCESS_KEY: some-secret-key
+  AWS_ACCESS_KEY_ID: some-access$\n"-key
```

Comment:
This addition covers this special characters case, but in the future no one will know why do we set secrets like this, with all of these special chars. We should somehow convey that we test this case in particular.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/258/commits/188f00266d32fefa3b75e146538c226cfaf40a2c